test-apps/presentation-test-app/src/frontend/components/properties-widget/PropertiesWidget.tsx

```diff
@@ -85,8 +85,11 @@ export function PropertiesWidget(props: Props) {
       onSelectedChanged: (index: React.SetStateAction<number>) => setActiveMatchIndex(index),
       resultCount: filteringResult.matchesCount,
     } : undefined;
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [filteringResult, filteringProvDataChanged]);
+  }, [filteringResult]);
+
+  const filteredTypes = React.useMemo(()=>{
```

Comment:
why do we need this? can't we just use `filteringResult?.filteredTypes` without `React.useMemo`?

Reply:
Yep. We don't :), fixed it